services/scraper/handler.go

```diff
@@ -0,0 +1,26 @@
+package scraper
+
+import (
+	"fmt"
+
+	"github.com/beego/beego/v2/core/logs"
+	"github.com/gocolly/colly/v2"
+)
+
+const currentBrowser = "Chrome/88.0.4324.182"
+const currentOs = "Macintosh; Intel Mac OS X 10_15_5"
+
+func onRequestHandler(request *colly.Request) {
+	userAgent := fmt.Sprintf("Mozilla/5.0 (%s) AppleWebKit/537.36 (KHTML, like Gecko) %s Safari/537.36", currentOs, currentBrowser)
```

Comment:
Pick a specific one browser as I don't need various screen for scraping 🙌
In different screen will build with different selector to scrape some content.

Reply:
@Lahphim Using a fixed user agent might cause the scraper to be blocked easier if we do a lot of scraping, it's easier to detect by the anti-scraping tools. But since we are not going to do that much intense scraping it might be fine to do this 🤔

Reply:
I agree with @carryall that randomizing the browser agents for each search will be needed when searching for more than one keyword within a few seconds. For now, this will work but surely it will need to be revisited either by having multiple browser agents or throttling the keyword searches.